src/api/app/views/webui/package/branch.html.haml

```diff
@@ -0,0 +1,28 @@
+:ruby
+  @pagetitle = "Branch #{@package}"
+
+.card.mb-3
+  .card-body
+    %h3= @pagetitle
+    - branch_project = User.session!.branch_project_name(@project)
+    .p-2
+      %strong Source:
+      %p
+        -# FIXME: get rid of this helper
+        = package_link(@package, short: true)
+      %strong Destination:
+      %p
+        -# FIXME: get rid of this helper
+        = project_or_package_link(project: branch_project, short: true)
+      = form_tag({ controller: :package, action: :create_branch }, method: :post) do
+        %details
+          %summary More options
+          .p-2
+            = hidden_field_tag :linked_project, @project.name
+            = hidden_field_tag :linked_package, @package.name
+            = render partial: 'webui/shared/package_branch_form', locals: { show_project_field: true,
```

Comment:
Using the grid also will make it possible to avoid overly long input fields.

Reply:
It's fixed, the applied grid keeps the input fields in place :)